lib/lita/adapters/slack/api.rb

```diff
@@ -33,10 +33,10 @@ def send_attachments(room_or_user, attachments)
         def send_messages(channel_id, messages)
           call_api(
             "chat.postMessage",
+            **config.postMessage,
```

Comment:
Are you sure `**` is available on all ruby versions supported by lita? It might be better to just do a `Hash#merge` for backward compatibility here.